docs/resources/credential_type.md

```diff
@@ -2,15 +2,15 @@
 page_title: "pingone_credential_type Resource - terraform-provider-pingone"
 subcategory: "Neo (Verifiable Credentials)"
 description: |-
-  Resource to create, read, and update the credential types used by compatible wallet applications.
-  ~> You must ensure that any fields used in the cardDesignTemplate are defined appropriately in metadata.fields or errors occur when you attempt to create a credential of that type.
+  Resource to create and manage the credential types used by compatible wallet applications.
+  ~> You must ensure that any fields used in the carddesigntemplate are defined appropriately in metadata.fields or errors occur when you attempt to create a credential of that type.
 ---
 
 # pingone_credential_type (Resource)
 
-Resource to create, read, and update the credential types used by compatible wallet applications.
+Resource to create and manage the credential types used by compatible wallet applications.
 
-~> You must ensure that any fields used in the cardDesignTemplate are defined appropriately in metadata.fields or errors occur when you attempt to create a credential of that type.
+~> You must ensure that any fields used in the card_design_template are defined appropriately in metadata.fields or errors occur when you attempt to create a credential of that type.
```

Comment:
Might be worth putting card_design_template in markdown quotes: `card_design_template`